src/minErr.js

```diff
@@ -34,6 +34,7 @@ function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
   return function() {
     var SKIP_INDEXES = 2;
+    var MAX_DEPTH = 1;
```

Comment:
There should be a way to customize this (I am not sure what it should be) and the default should be a little higher imo (maybe 3 or 4).